Xamarin.Forms.Core/Span.cs

```diff
@@ -1,145 +1,101 @@
 using System;
-using System.ComponentModel;
-using System.Runtime.CompilerServices;
 using Xamarin.Forms.Internals;
 
 namespace Xamarin.Forms
 {
 	[ContentProperty("Text")]
-	public sealed class Span : INotifyPropertyChanged, IFontElement
+	public sealed class Span : Element, IFontElement
 	{
-		class BindableSpan : BindableObject, IFontElement
+		internal readonly MergedStyle _mergedStyle;
+
+		public Span()
 		{
-			Span _span;
-			public BindableSpan(Span span)
```

Comment:
Now that the Span is bindable, these properties were pulled out of here and available directly on the span, instead of providing a relay.